### DISCUSSION TITLE
Running mlflow ui in sagemaker does not work

### DISCUSSION THREAD

**Discussion by Jason-F** [mlflow/mlflow#5902] [Q&A] [answered]
Running mlflow ui in the sagmaker terminal returns:

`[2022-05-19 07:44:04 +0000] [4411] [INFO] Starting gunicorn 20.1.0

[2022-05-19 07:44:04 +0000] [4411] [INFO] Listening at: http://127.0.0.1:5000 (4411)

[2022-05-19 07:44:04 +0000] [4411] [INFO] Using worker: sync

[2022-05-19 07:44:04 +0000] [4414] [INFO] Booting worker with pid: 4414`

Opening http://127.0.0.1:5000 in the web browser does not work, you get "This site can’t be reached127.0.0.1 refused to connect."

How do I run mlflow ui in sagemaker? the Tracking uri is set withint the sagemaker directory

**Comment by dbczumar** [comment, marked as answer]
Hi @Jason-F, unfortunately, this appears to be a limitation of the SageMaker terminal, rather than a limitation of MLflow. I would suggest filing a ticket with SageMaker support. Thank you for using MLflow!
